src/paste.rs

```diff
@@ -233,7 +245,10 @@ impl Paste {
                     .to_string());
             }
         }
-        Ok(self.store_file(file_name, expiry_date, &config)?)
+
+        let file_name = process_filename_for_storage(file_name, &config);
```

Comment:
This is not needed, because this function is called in `store_file`. If you left it here, the same funtion would be called twice.

Reply:
Thank you for highlighting that. I've made the necessary correction and updated the commit.